Remotion/ObjectBinding/Web.Development.WebTesting.TestSite.NetFramework/ObjectBinding.Web.Development.WebTesting.TestSite.NetFramework.csproj

```diff
@@ -138,6 +138,7 @@
   </ItemGroup>
   <ItemGroup>
     <Content Include="Global.asax" />
+    <Content Include="UrlMapping.xml" />
```

Comment:
Moving the UrlMapping file isn't something I'm happy with. it is a duplication of code.